string message
float32 time
float32 vel_lin_x
float32 vel_lin_y
float32 vel_lin_z
float32 vel_ang_x
float32 vel_ang_y
float32 vel_ang_z